x/valset/keeper/keep_alive.go

```diff
@@ -128,28 +141,39 @@ func (k Keeper) UpdateGracePeriod(ctx sdk.Context) {
 		lookup[string(v)] = struct{}{}
 	}
 
-	vals := slice.Map(k.GetUnjailedValidators(ctx), func(i stakingtypes.ValidatorI) []byte {
-		return i.GetOperator()
+	vals, err := slice.MapErr(k.GetUnjailedValidators(ctx), func(i stakingtypes.ValidatorI) ([]byte, error) {
+		bz, err := k.addressCodec.StringToBytes(i.GetOperator())
+		if err != nil {
+			return nil, err
+		}
+		return bz, nil
```

Comment:
What about extracting this into a util function? Seems to be used super frequently. 

What possible errors might occur?

Reply:
It makes sense. If `i.GetOperator()` returns a valid string always then we can remove the error handler in all the 5 occurrences.

Reply:
pushed the above mentioned two changes @byte-bandit